### ISSUE TITLE
Inspector pane is empty if you enable FireQuery while having Firebug 3.0b1 enabled

### ISSUE THREAD

**Issue by darkred** [firebug/firequery#34]
_Using FireQuery 2.0.1 with Firebug 3.0.0 beta 1 in Nightly x64 2015-10-30._

When I start Nightly, having only Firebug enabled, and either 
- press the toolbar button of Firebug or 
- rightclick on an element and 'press Inspect Element'

then Inspector pane works ok, i.e. the HTML and CSS panes show info on the selected element.

If now I enable FireQuery, then Inspector panes appear empty: 
![](http://i.imgur.com/j9xgUB0h.jpg)
and clicking the icon to "Pick an element from the page" does nothing.

When this occurs, disabling both FireBug + FireQuery (having now no other addons enabled) doesn't fix the problem - I have to restart Nightly (with FireQuery disabled) for Inspector to work again.

Here's what's shown in Browser Console shows when this occurs:

```
Error writing response to: getWalker                                protocol.js:1025
<unavailable>                                                       protocol.js:907
Protocol error (unknownError): InternalError: too much recursion    Promise-backend.js:937
```

<br/>
This appears every single time on my everyday FF profile _(for some reason I couldn't reproduce it in a clean profile, though)._

**Comment by janodvarko** [comment]
I can't reproduce the problem. Here is what I am doing:

1) Install FireQuery, disable, restart Firefox
2) Load: http://softwareishard.com/firequery/test1/index.html
3) Open the Toolbox (F12), select the Inspector panel
4) Open about:addons in another browser tab
5) Enable FireQuery
6) Go back to the first tab, check the Inspector panel. It is _not_ empty.

(I am running the latest Nightly)

Are you able to reproduce the problem using these steps?
What should I do differently?
What is your OS?

Honza

**Comment by darkred** [comment]
I reproduced your steps and yes, the inspector is not empty.

_(BTW, clicking on the envelope icon in the Inspector works ok (display a popup with details -as expected-),
but, clicking the icon in Console does not show info in DOM properties : 
instead this is shown in Browser Console (in both e10s and non-e10s windows):_ 

```
TypeError: VariablesView is undefined            jquery-renderer.js:148:7
```

---

My OS is win10x64 build 10240.
My other addons are
- Greasemonkey 2015.10.23.nightly  (with quite a few userscripts installed).
- AdBlock Plus beta 2.6.11.3994 beta and 
- Element Hiding Helper for ABP 1.3.4.510 beta 

_(For what it's worth, I also have DownThemAll 2.0.18 and ImageHost Grabber 1.7.0.3 but they are disabled all the time)._

I've tried to duplicate my profile in a new one,
_(i.e. installing there all the above addons and userscripts (the `gm_scripts` folder),
all bookmarks, pinning some tabs, and grouping some tabs in panorama)_,
but still the issue is not occuring.

---

Also. another thing I noticed in Browser Console:
whenever this occurs,
apart from the the 3 error entries I gave in the OP,
a second later, these extra 2 entries appear:
(I've added seperators to make them somehow more easily read): 

```
Error writing response to: getWalker                               protocol.js:1025
<unavailable>                                                      protocol.js:907
Protocol error (unknownError): InternalError: too much recursion   Promise-backend.js:937


------------------------------------------------------------------------------------------------------

A promise chain failed to handle a rejection. Did you forget to '.catch', or did you forget to 'return'?                                nsContextMenu.js:587:0
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Date: Mon Nov 02 2015 19:09:00 GMT+0200 (GTB Standard Time)
Full Message: TypeError: inspector is undefined
Full Stack: CM_inspectNode/<@chrome://browser/content/nsContextMenu.js:587:9
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
DevTools.prototype.showToolbox/<@resource://devtools/client/framework/gDevTools.jsm:423:9
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
TaskImpl_handleException@resource://gre/modules/Task.jsm:448:5
TaskImpl_run@resource://gre/modules/Task.jsm:326:11
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:937:21
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
Front<.onPacket/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1234:9
DevTools RDP*Front<.request@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1178:7
frontProto/</proto[name]@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1338:14
exports.InspectorFront<.getWalker<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/actors/inspector.js:3780:12
Toolbox.prototype.initInspector/this._initInspector<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1792:30
TaskImpl_run@resource://gre/modules/Task.jsm:314:40
TaskImpl@resource://gre/modules/Task.jsm:275:3
createAsyncFunction/asyncFunction@resource://gre/modules/Task.jsm:249:14
Task_spawn@resource://gre/modules/Task.jsm:164:12
Toolbox.prototype.initInspector@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1790:29
Toolbox.prototype.loadTool@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1165:14
Toolbox.prototype.selectTool@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1357:12
Toolbox.prototype.open/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:408:13
TaskImpl_run@resource://gre/modules/Task.jsm:314:40
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
Toolbox.prototype.open/</<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:356:36

------------------------------------------------------------------------------------------------------


A promise chain failed to handle a rejection. Did you forget to '.catch', or did you forget to 'return'?                              <unknown>
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Date: Mon Nov 02 2015 19:09:00 GMT+0200 (GTB Standard Time)
Full Message: Protocol error (unknownError): InternalError: too much recursion
Full Stack: JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: PendingErrors.register :: line 191
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.completePromise :: line 715
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: Handler.prototype.process :: line 970
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.walkerLoop :: line 813
```

**Comment by janodvarko** [comment]
Can you try this test XPI:
janodvarko.cz/temp/firequery/firequery-2.0.1.34.xpi

Does it solve the problem?
Honza

**Comment by darkred** [comment]
I tried it and unfortunately, it doesn't fix it. (and I get the exactly same errors in Browser Console)

**Comment by janodvarko** [comment]
@darkred: Do you still see the
TypeError: VariablesView is undefined            jquery-renderer.js:148:7
error?

Honza

**Comment by darkred** [comment]
~~All is working ok now. Both issues are solved!
Thanks a lot for fixing them!~~

Just one last thing to mention:
I've noticed that, though fireQuery is restartless,
if you disable it, it still works until you restart Firefox.
~~The same applies when you enable it: a FF restart is needed for it to work.~~

**Comment by janodvarko** [comment]
> All is working ok now. Both issues are solved!
> Thanks a lot for fixing them!

Excellent

> Just one last thing to mention:
> I've noticed that, though fireQuery is restartless,
> if you disable it, it still works until you restart Firefox.
> The same applies when you enable it: a FF restart is needed for it to work.

Can you please file a new issue for this plus more info about what it means that "it still works".

I am closing this one.

Thanks for the cooperation!

Honza

**Comment by darkred** [comment]
I spoke too soon... Sorry, but I have to re-open it...
(and this issue is related to bug #25)

I still can't reproduce it on a clean profile, but on my everyday profile it goes like this:

with fireQuery closed (fireBug always enabled), Inspector works **ok all the time**.
But, 
- if you launch Firefox having (fireBug always enabled and) fireQuery disabled, 
  and you have one tab with some page with Inspector open (focused on an element), 
  you enable fireQuery, then Inspector will work ok **until you close Inspector**(DevTools). 
  If then you re-open Inspector(DevTools, or rightclick an element|'Inspect Element'), then Inspector will be **totally** empty (as in the screenshot in my OP).
- Then, if you close and re-launch Firefox, and open Inspector (now having fireQuery enabled), 
  then Inspector (the HTML pane), will show the page code for a split of a second, and then it will be **almost empty**, i.e. will show an almost empty `<head></head>` and an almost empty `<body></body>`
  Screenshot while trying to use Inspector in this page: ![](https://i.imgur.com/HaNBs5s.jpg)
  That's why I mistakenly thought that it was fixed....
- On consequent relaunches of FIrefox, the Inspector's HTML pane  will be **totally empty**.

_Sorry for the confusion, but it's been proven very difficult to correctly describe and reproduce this issue..._

---

But, the `TypeError: VariablesView is undefined jquery-renderer.js:148:7
error` is fixed.

---

Here are messages in Browser Console before relaunching FIrefox (i.e. with Inspector/HTML pane **totally** empty): -they are the same as in [this post above](https://github.com/firebug/firequery/issues/34#issuecomment-153089868)-

```
Error writing response to: getWalker protocol.js:1025
<unavailable> protocol.js:907
Protocol error (unknownError): InternalError: too much recursion Promise-backend.js:937
A promise chain failed to handle a rejection. Did you forget to '.catch', or did you forget to 'return'?
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Date: Wed Nov 04 2015 03:16:08 GMT+0200 (GTB Standard Time)
Full Message: Protocol error (unknownError): InternalError: too much recursion
Full Stack: JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: PendingErrors.register :: line 191
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.completePromise :: line 715
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: Handler.prototype.process :: line 970
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.walkerLoop :: line 813 <unknown>
A promise chain failed to handle a rejection. Did you forget to '.catch', or did you forget to 'return'?
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Date: Wed Nov 04 2015 03:16:07 GMT+0200 (GTB Standard Time)
Full Message: TypeError: inspector is undefined
Full Stack: CM_inspectNode/<@chrome://browser/content/nsContextMenu.js:587:9
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
DevTools.prototype.showToolbox/<@resource://devtools/client/framework/gDevTools.jsm:423:9
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
TaskImpl_handleException@resource://gre/modules/Task.jsm:448:5
TaskImpl_run@resource://gre/modules/Task.jsm:326:11
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:937:21
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
Front<.onPacket/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1234:9
DevTools RDP*Front<.request@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1178:7
frontProto/</proto[name]@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/protocol.js:1338:14
exports.InspectorFront<.getWalker<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/actors/inspector.js:3780:12
Toolbox.prototype.initInspector/this._initInspector<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1792:30
TaskImpl_run@resource://gre/modules/Task.jsm:314:40
TaskImpl@resource://gre/modules/Task.jsm:275:3
createAsyncFunction/asyncFunction@resource://gre/modules/Task.jsm:249:14
Task_spawn@resource://gre/modules/Task.jsm:164:12
Toolbox.prototype.initInspector@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1790:29
Toolbox.prototype.loadTool@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1165:14
Toolbox.prototype.selectTool@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:1357:12
Toolbox.prototype.open/<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:408:13
TaskImpl_run@resource://gre/modules/Task.jsm:314:40
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:934:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:813:7
Promise*this.PromiseWalker.scheduleWalkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:744:11
this.PromiseWalker.schedulePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:776:7
this.PromiseWalker.completePromise@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:711:7
Toolbox.prototype.open/</<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/client/framework/toolbox.js:356:36
 nsContextMenu.js:587:0
```

---

and here is after relaunch (i.e. with Inspector/HTML pane **almost** empty):

```
Use of nsIFile in content process is deprecated. filesystem.js:40:9
Firebug.next ERROR: No search box! inspector-overlay.js:209:0
Firebug.next ERROR: No search box! inspector-overlay.js:209:0
Firebug.next ERROR: No search box! inspector-overlay.js:209:0
Using //@ to indicate sourceURL pragmas is deprecated. Use //# instead db1fae0d-5dff-4854-9c70-ff5799f7d546:6797:0
"DevToolsUtils.dbg_assert is deprecated! Use DevToolsUtils.assert instead!
dbg_assert@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/shared/DevToolsUtils.js:449:13
TabActor.prototype.sources@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/actors/webbrowser.js:897:7
ThreadActor.prototype.sources@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/actors/script.js:509:5
ThreadActor.prototype.onAttach@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/actors/script.js:627:5
DSC_onPacket@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/server/main.js:1606:15
ChildDebuggerTransport.prototype.receiveMessage@resource://gre/modules/commonjs/toolkit/loader.js -> resource://devtools/shared/transport/transport.js:742:5
" DevToolsUtils.js:452
actor.js; expectState EXCEPTION Error: call to eval() blocked by CSP firequery-actor.js:84:0
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src https://assets-cdn.github.com"). firequery-actor.js:335:0
Error writing response to: querySelector protocol.js:1025
<unavailable> protocol.js:907
Protocol error (unknownError): InternalError: too much recursion Promise-backend.js:937
```

**Comment by janodvarko** [comment]
Thanks for the analysis! 

> (and this issue is related to bug #25)

Yeah, I spent some time on it too and it would be great to have both fixed.

So, if I understand correctly, the problem is gone if Firebug 3 beta 1 is disabled?

Honza

**Comment by darkred** [comment]
No, even with Firebug 3 beta 1 (and all other addons) disabled, 
i.e **having only fireQuery enabled while restarting Firefox**, 
the problem exists, i.e. Inspector is "almost empty " (when you rightclick an element|Inspect Element):
![](http://i.imgur.com/WfqsevV.jpg)

What only works is clicking an envelope icon in Console (it show info in DOM properties, as expected).

Note: this is not the same to the procedure you gave to me in your [first comment](https://github.com/firebug/firequery/issues/34#issuecomment-153007547) 
where you asked me to **launch Firefox with fireQuery disabled**, then open Inspector, and at the end enable fireQuery,
because in that case, Inspector doesn't become empty when you enable fireQuery,
but the point is that fireQuery **is not applied to Inspector** (there are no envelope icons in either Inspector or Console)
until you close and reopen Inspector. 
At that point fireQuery should be applied to the Inspector; 
but, instead, unfortunately, Inspector is empty.


PS. I'm referring to my everyday profile.

**Comment by darkred** [comment]
Please give me a few minutes to answer your question/

**Comment by darkred** [comment]
> So, if I understand correctly, the problem is gone if Firebug 3 beta 1 is disabled? 

No, even with Firebug 3 beta 1 (and all other addons) disabled 
i.e. Having only fireQuery enabled while restarting Firefox**,  the problem remains:

---

If you **launch Firefox with fireQuery enabled**: and visit the test page and open Inspector(rightclick an element|Inspect Element), 
it will show the page code for a split of a second, and then it becomes **totally empty** 
What only works is -after clicking the "Log data" button- is that the entries in Console have an envelope icon  (and clicking it, shows info in DOM properties, as expected).

---

If you **launch Firefox with fireQuery disabled**, then there are 2 sub-cases: 
- if you **first enable fireQuery in about:addons and then open Inspector**, 
  then Inspector is **almost empty** : 
  ![](http://i.imgur.com/jrVmL0i.jpg)
  and the "Pick Element from the page" arrow icon can't select anything but the 3 existing tags (html,head,body).
- **If you first open Inspector and then enable fireQuery in about:addons**, 
  then Inspector keeps working ok, but the fireQuery is not yet applied: 
  there's no envelope icon in neither Inspector or Console. 
  If now you close and reopen Inspector, then Inspector is **totally empty**: 
  ![](http://i.imgur.com/6JAjv3R.jpg)

---

PS. I'm referring to my everyday profile, and doing the tests in the test page http://softwareishard.com/firequery/test1/index.html.

<br/>

_On the other hand, In a clean profile, fireQuery works ok, either enabling it after FF launch, or having it enabled at FF launch. Also, in both cases, Inspector then is not empty, the arrow icon works ok in all cases, and clicking on an envelope in both Inspector and Console works ok._*

**Comment by janodvarko** [comment]
Would it be possible to compare your profiles (everyday with a clean profile) and check what difference could cause the problem? Interesting could be to see differences in prefs.js file (preferences).

Honza

**Comment by darkred** [comment]
As I've previously written
> I've tried to duplicate my profile in a new one,
> *(i.e. installing there all the above addons and userscripts (the `gm_scripts` folder),
> all bookmarks, pinning some tabs, and grouping some tabs in panorama)*,

i.e. I've tried to duplicate it as close as possible, but the issue is not reproducible.
It's probably some parameter I miss, but that 's the best I could.

The only two about:config entries that I've changed in my everyday profile is enabling: 
`devtools.jsonview.enabled` and 
`devtools.memory.enabled` 
but disabling them(and restarting FF) doesn't make any difference.

**Comment by darkred** [comment]
As I've previously written

> I've tried to duplicate my profile in a new one,
> _(i.e. installing there all the above addons and userscripts (the `gm_scripts` folder),
> all bookmarks, pinning some tabs, and grouping some tabs in panorama)_,

i.e. I've tried to duplicate it as close as possible, but the issue is not reproducible.
It's probably some parameter I miss, but that 's the best I could.

The only two about:config entries that I've changed in my everyday profile are enabling: 
`devtools.jsonview.enabled` and 
`devtools.memory.enabled` 
but disabling them(and restarting FF) doesn't make any difference.

Here is a diff of the two prefs.js _(there wasn't any way to export the diff apart from this screenshot)_.
https://i.imgur.com/5dv9rmR.jpg
I couldn't find any relevant entry to toggle (and they are too many).

**Comment by darkred** [comment]
I've just edited my [previous comment](https://github.com/firebug/firequery/issues/34#issuecomment-153816225) to make it more accurate.

**Comment by SamB** [comment]
@darkred: Do you somehow not have access to unix diff? If so, "`git diff --no-index`" might be useful...

**Comment by darkred** [comment]
It's been quite a long time since my last comment.
In the meanwhile I have made a fresh Firefox profile _(I don't have a backup of that from Nov 2015)_.

I haven't encountered the issue in my clean Firefox 48 profile since then,
nor in a clean Nightly 51 x64 profile.

There's nothing more I can contribute to this. so I'm closing it.

Thanks for your time, Jan.
